install/startup.sh

```diff
@@ -2,38 +2,37 @@
 
 echo "Wait for Kubernetes context to be set and ready"
 
```

Comment:
which namespace will this use? default or flippy?

Reply:
Whatever kubectx has set to. Service account or kubeconfig sets namespace out of box.
Switched to kubeconfig specified namespace because it will be free form based upon service account or downloaded kubeconfig. Idea behind is does pod have access to k8s API or not.